docs/api.md

```diff
@@ -0,0 +1,366 @@
+[Documentation Home](./README.md)
+
+# API
+
+- [Generate a Prompt](#generate-a-prompt)
+- [Create a Model](#create-a-model)
+- [List Local Models](#list-local-models)
+- [Copy a Model](#copy-a-model)
+- [Delete a Model](#delete-a-model)
+- [Pull a Model](#pull-a-model)
+
+## Generate a Prompt
+
+**POST /api/generate**
+
+### Description
+
+**Generate** is the main endpoint that you will use when working with Ollama. This is used to generate a response to a prompt sent to a model.
+
+### Request
+
+The **Generate** endpoint takes a JSON object with the following fields:
+
+```JSON
+{
+  "model": "modelname",
```

Comment:
Somewhere in this docs it should mention the modelname format

Reply:
can you clarify